docs/topics/proc_configuring-your-application-to-use-spring-boot.adoc

```diff
@@ -47,6 +47,50 @@ Include the following properties in your `pom.xml` file to track the version of
 ----
 --
 
+. Specify the repositories containing the {SpringBoot} Maven plugin and the {SpringBoot} dependencies you are using in your application.
++
+[source,xml,subs="attributes+"]
+----
+<project>
+  ...
+  <!-- Specify the repositories containing {Spring Boot} artifacts. -->
+  <repositories>
+    <repository>
+      <id>redhat-ga</id>
+      <name>Red Hat GA Repository</name>
+      <url>https://maven.repository.redhat.com/ga/</url>
+    </repository>
+  </repositories>
+
+  <!-- Specify the repositories containing the plugins used to execute the build of your application. -->
+  <pluginRepositories>
+    <pluginRepository>
+      <id>redhat-ga</id>
+      <name>Red Hat GA Repository</name>
+      <url>https://maven.repository.redhat.com/ga/</url>
+    </pluginRepository>
+  </pluginRepositories>
+  ...
+</project>
+
+<repositories>
+  <repository>
+    <id>central</id>
+    <name>Central Repository</name>
+    <url>http://repo.maven.apache.org/maven2</url>
+  </repository>
+</repositories>
+
+<pluginRepositories>
+  <pluginRepository>
+    <id>central</id>
+    <name>Central Repository</name>
+    <url>http://repo.maven.apache.org/maven2</url>
+  </pluginRepository>
+</pluginRepositories>
```

Comment:
Why exactly do you configure Central, which is configured in Maven by default, and especially outside of `<project>`?

Reply:
RE: why central: [According to information provided by JohnC](https://access.redhat.com/articles/3710221#comment-1388291), we ship to Spring Boot bits to Maven Central rather than MRRC (to circumvent legal constraints on using the name _Spring Boot_ for the Red Hat Productized release of SB).
I wanted to ensure we are not pointing to MRRC when the bits are in Maven central.
Since Central is configured by default, I'll remove this from step form the procedure.

Reply:
Yea, Maven always downloads from Central, unless you take drastic measures to prevent that (and those can't be taken in `pom.xml` to the best of my knowledge).

Reply:
@Ladicek Thank you for the clarification:) I'll remove the step, then. No sense in introducing this amount of redundancy.